trigger_circleci_pipeline.sh

```diff
@@ -99,3 +99,27 @@ for workflow_id in $pipeline_workflows_response
 do
   echo "${CIRCLE_WORKFLOW_URL_BASE}/${workflow_id}"
 done
+
+if [ "${SHOULD_WAIT}" = "true" ]; then
+    echo "Waiting for workflows to finish"
```

Comment:
```suggestion
    echo "Waiting for workflow to finish"
```

Reply:
I think that the result returned by the command is a list of workflows, shouldn't that be workflows?